.github/workflows/build.yml

```
@@ -21,27 +21,27 @@ jobs:
         ruby-version: ${{ matrix.ruby }}
         bundler-cache: true
     - name: Bundle install
-      run: bundle install --jobs 4
+      run: bundle install
     - name: Specs with coverage
       if: ${{ matrix.ruby == 2.7 }}
-      run: bundle exec rake
+      run: "RUBYOPT='--enable=frozen-string-literal --debug=frozen-string-literal' bundle exec rake"
       env:
         COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
         CI_PULL_REQUEST: ${{ github.event.number }}
         CI_BRANCH: ${{ github.head_ref }}
     - name: Specs
       if: ${{ matrix.ruby != 2.7 }}
-      run: bundle exec rake
+      run: "RUBYOPT='--enable=frozen-string-literal --debug=frozen-string-literal' bundle exec rake"
   Rubocop:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set up Ruby 2.7
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 2.7
```

On pourrait enlever le ruby-version explicite et laisser le fichier `.ruby-version` le définir.

```suggestion
      - name: Set up Ruby
        uses: ruby/setup-ruby@v1
        with:
```